codecov.yml

```diff
@@ -5,3 +5,14 @@ comment:
   layout: "reach,diff,flags,tree"
   behavior: default
   require_changes: no
+
+coverage:
+  status:
+    project:
+      default:
+        threshold: 0.25%
+    patch: off
```

Comment:
What does this do? I couldn't find documentation.

Reply:
`patch`? I was copied from SwiftInspector, which introduced this in https://github.com/fdiaz/SwiftInspector/pull/132